src/configure/configurers/provisioningConfigurer.ts

```diff
@@ -131,9 +134,11 @@ export class ProvisioningConfigurer implements IProvisioningConfigurer {
                 });
         } else {
             telemetryHelper.setTelemetry(TelemetryKeys.PipelineDiscarded, 'true');
+            await this.moveWorkflowFilesToRepoLocally();
             throw new UserCancelledError(Messages.operationCancelled);
         }
 
+        fse.removeSync(this.tmpDirectoryPath);
```

Comment:
Will this also remove the files present in temp dir?

Reply:
Yes